packages/offix-client/src/OfflineClient.ts

```diff
@@ -121,15 +103,24 @@ export class OfflineClient {
         debug: false
       });
     }
+
+    this.offix = new Offix({
```

Comment:
I do not like name of this. Mentioned it in different PR. Offix is vague. OffixScheduler works much better.